mw-webapp/src/styles/_variables.scss

```diff
@@ -12,4 +12,7 @@ $link-active: rgb(255, 149, 0);
 $header-background: rgb(85, 122, 149);
 
 // Title
-$title-padding: 10px;
\ No newline at end of file
+$title-padding: 10px;
+
+//Select
+$select-font-size: 1em;
```

Comment:
I think 1em(16px) is the default font size? I don't see the point of setting it then.

Reply:
done